components/post_view/post_message_view/post_message_view.test.tsx

```diff
@@ -49,32 +49,35 @@ describe('components/post_view/PostAttachment', () => {
     });
 
     test('should match snapshot, on deleted post', () => {
-        const props = {...baseProps, post: {...post, state: Posts.POST_DELETED}};
+        const props = {...baseProps, post: {...post, state: Posts.POST_DELETED as 'DELETED'}};
         const wrapper = shallow(<PostMessageView {...props}/>);
+        const instance = wrapper.instance() as PostMessageView;
 
         expect(wrapper).toMatchSnapshot();
-        expect(wrapper.instance().renderDeletedPost()).toMatchSnapshot();
+        expect(instance.renderDeletedPost()).toMatchSnapshot();
     });
 
     test('should match snapshot, on edited post', () => {
         const props = {...baseProps, post: {...post, edit_at: 1}};
         const wrapper = shallow(<PostMessageView {...props}/>);
+        const instance = wrapper.instance() as PostMessageView;
 
         expect(wrapper).toMatchSnapshot();
-        expect(wrapper.instance().renderEditedIndicator()).toMatchSnapshot();
+        expect(instance.renderEditedIndicator()).toMatchSnapshot();
     });
 
     test('should match snapshot, on ephemeral post', () => {
-        const props = {...baseProps, post: {...post, type: Posts.POST_TYPES.EPHEMERAL}};
+        const props = {...baseProps, post: {...post, type: Posts.POST_TYPES.EPHEMERAL as PostType}};
         const wrapper = shallow(<PostMessageView {...props}/>);
 
         expect(wrapper).toMatchSnapshot();
     });
 
     test('should match checkOverflow state on handleHeightReceived change', () => {
         const wrapper = shallow(<PostMessageView {...baseProps}/>);
-        const instance = wrapper.instance();
-        instance.checkOverflow = jest.fn();
+        const instance = wrapper.instance() as PostMessageView;
+
+        //instance.checkOverflow = jest.fn();
```

Comment:
This is clearly not needed, looks like some obsolete code, so we can remove the comment also.

Reply:
Comment removed.